src/plugins/es_ui_shared/public/index.ts

```diff
@@ -47,6 +47,18 @@ export {
   expandLiteralStrings,
 } from './console_lang';
 
+export {
```

Comment:
To avoid naming collisions, the idea was not to export from the top-level `index.ts`. For example, here we might probably export another `Error` interface in a different context.

Reply:
Right, that is something I am meaning to address properly in future. A smart refactor tool should do the heavy lifting for us when we move that folder.

I created this issue for discussion though: https://github.com/elastic/kibana/issues/64052

Reply:
Awesome thanks for opening the issue! 👍